packages/story-editor/src/app/story/useStoryReducer/reducers/copySelectedElement.js

```diff
@@ -50,21 +50,9 @@ function copySelectedElement(state) {
     return state;
   }
 
-  // Do nothing if no elementId
   const elementId = state.selection[0];
-  if (!elementId) {
-    return state;
-  }
-
   const page = state.pages.find(({ id }) => id === state.current);
-
   const elementIndex = page.elements.findIndex(({ id }) => id === elementId);
```

Comment:
Selection can never have invalid IDs. That is an invariant maintained by the entire system, so no need to check for that here.